cts/cts-cli.in

```diff
@@ -920,6 +937,8 @@ for t in $tests; do
         -e 's|^/tmp/cts-cli\.validity\.bad.xml\.[^:]*:|validity.bad.xml:|'\
         -e 's/^Entity: line [0-9][0-9]*: //'\
         -e 's/\(validation ([0-9][0-9]* of \)[0-9][0-9]*\().*\)/\1X\2/' \
+        -e 's/^Migration will take effect until: .*/Migration will take effect until:/' \
+        -e 's/ end=\"[-: 0123456789]\+Z\?\"/ end=\"\"/' \
```

Comment:
The FreeBSD tests are failing due to BSD sed not supporting + ... sadly, we have to use e.g. XX* instead of X+, although in this case just using * instead of + should be fine

Reply:
BSD sed supports + if you give it the -E option, meaning use extended regexes instead of basic ones.  It looks like Linux sed supports that option too, though I think it's probably just a no-op.  I'll push a new version of this patch to this PR here in just a minute.

Reply:
... turns out to be a bad idea, detailed in
https://github.com/ClusterLabs/pacemaker/pull/1677/commits/286569c1a2359a97ad33f77b3b6b63b04d520b76